packages/components/src/internal/components/search/SampleFinderManageViewsModal.tsx

```diff
@@ -65,6 +65,19 @@ export const SampleFinderManageViewsModal: FC<Props> = memo(props => {
             return;
         }
 
+        const existingViews = await api.samples.loadFinderSearches();
+        let duplicate = false;
+        existingViews.forEach(v => {
+            if (v.reportName.toLowerCase() === newName.toLowerCase()) {
```

Comment:
manual testing issue: I was able to get a duplicate name by putting some spaces before the name. Looks like we might just need to add a .trim() to the newName here.